### DISCUSSION TITLE
Team

### DISCUSSION THREAD

**Discussion by Bashamega** [cytrinc/bark-coding#184] [Admin General]
Hello:)
I am a part of the team and i want the authority like vercel teams, gmails etc

**Comment by raynepaws** [comment]
only mariocraft can access vercel, and only i can manage the email. sorry if this sounds rude, but why does it matter?

if you would like to be added to the list of people that get emails to bark.github.io@gmail.com forwarded to them, just give me your email and i can add it to the list

**Reply by Bashamega** [reply]
Sure,
adam.webtools@gmail.com

**Reply by raynepaws** [reply]
i'll add this tomorrow, i'm gonna get some rest rn

**Reply by mariocraft987** [reply]
@qwertyy-dev has the [bark.github.io@gmail.com](mailto:bark.github.io@gmail.com)

**Reply by raynepaws** [reply]
@Bashamega @BwnnyRxbbit @mariocraft987 you should get confirmation emails in your inboxes. you have to accept so i can set up automatic forwarding of emails

**Reply by Bashamega** [reply]
> @Bashamega @BwnnyRxbbit @mariocraft987 you should get confirmation emails in your inboxes. you have to accept so i can set up automatic forwarding of emails

Done

**Reply by raynepaws** [reply]
> Done

added! you'll receive forwarded emails from now on
